waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -64,15 +63,15 @@ method put*(db: WakuMessageStore, cursor: Index, message: WakuMessage, pubsubTop
   ##     echo "error"
   ## 
   let prepare = db.database.prepareStmt(
-    "INSERT INTO " & TABLE_TITLE & " (id, receiverTimestamp, contentTopic, payload, pubsubTopic, version, senderTimestamp) VALUES (?, ?, ?, ?, ?, ?, ?);",
-    (seq[byte], float64, seq[byte], seq[byte], seq[byte], int64, float64),
+    "INSERT INTO " & TABLE_TITLE & " (id, contentTopic, payload, pubsubTopic, version, senderTimestamp) VALUES (?, ?, ?, ?, ?, ?);",
```

Comment:
I'd keep both around, and then just have one that is defaulted to index (this can be sender if we think it is a more common use case)

Reply:
Sure, I can keep the receiver timestamp column inside the DB, however, we will have no code to generate the receiver timestamp of the incoming messages. The receiver timestamp used to be part of computing the index of the received waku message which is now replaced by the sender timestamp.
The reason I have removed the receiver timestamp entirely was to avoid subsequent confusion.
What would you see as the use-case for the receiver timestamp?

FYI, we had also a conversation during EU/NA call about the potential use-case of the receiver timestamp to spot messages with falsified timestamps, However, such timestamp validation can be done simply by comparing the receiver current time against the timestamp of the incoming message without having to store the receiver timestamp locally.